kvdb/postgres/db_test.go

```diff
@@ -14,33 +14,39 @@ import (
 
 // TestInterface performs all interfaces tests for this database driver.
 func TestInterface(t *testing.T) {
-	f := NewFixture(t)
-	defer f.Cleanup()
+	err := StartEmbeddedPostgres()
+	require.NoError(t, err)
+	defer StopEmbeddedPostgres()
+
+	f, err := NewFixture("")
+	require.NoError(t, err)
 
 	// dbType is the database type name for this driver.
 	const dbType = "postgres"
 
 	ctx := context.Background()
 	cfg := &Config{
-		Dsn: testDsn,
+		Dsn: f.Dsn,
 	}
 
 	walletdbtest.TestInterface(t, dbType, ctx, cfg, prefix)
 }
 
 // TestPanic tests recovery from panic conditions.
 func TestPanic(t *testing.T) {
-	f := NewFixture(t)
-	defer f.Cleanup()
+	err := StartEmbeddedPostgres()
```

Comment:
Something I experimented with in another project is using `https://github.com/ory/dockertest`. So far I like it somewhat better as it's possible to test with a specific backend version easily and also is more portable afaict. Although a tad bit more heavy than the embedded postgres.. Maybe consider taking a look.

Reply:
Interesting, but for me it is out of scope for this PR.